tasks/testwasher.py

```diff
@@ -88,3 +102,39 @@ def process_module_results(self, module_results: List[ModuleTestResult]):
             print(failed_tests_string)
 
         return should_succeed
+
+    def is_known_flaky_test(self, failing_test, known_flaky_tests, known_flaky_tests_parents):
```

Comment:
Technically we could compute `known_flaky_tests_parents` from `known_flaky_tests`. I did not do it to avoid computing it on very test name. We can compute it once for each package. We could say that we do not care about the performance here and compute the `known_flaky_tests_parents` every time the method is called

Reply:
I find it cleaner to have the ancestors as an input to this function, building it per package seems fine